src/dao/hooks/useCreateMultiSigProposal.tsx

```diff
@@ -1,5 +1,5 @@
 import { useMutation, useQueryClient } from "react-query";
-import { TransactionResponse, TokenId, AccountId } from "@hashgraph/sdk";
+import { TransactionResponse, TokenId, AccountId, TokenType } from "@hashgraph/sdk";
 import { DAOMutations, DAOQueries } from "./types";
```

Comment:
We could create a TokenType enum in constants and use it in single place everywhere.

Reply:
Will Move TokenType to Service with Enhancement PR, along with UI changes and MirrorNodeDecodedProposalEvent Interface Changes.